src/components/ZetkinQuery.stories.tsx

```diff
@@ -0,0 +1,47 @@
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+import { LinearProgress, Typography } from '@material-ui/core';
+
+import mockUseQueryResult from 'utils/testing/mocks/mockUseQueryResult';
+import ZetkinQuery from './ZetkinQuery';
+
+export default {
+  component: ZetkinQuery,
+  title: 'Atoms/ZetkinQuery',
+} as ComponentMeta<typeof ZetkinQuery>;
+
+const Template: ComponentStory<typeof ZetkinQuery> = (args) => (
+  <ZetkinQuery
+    errorIndicator={args.errorIndicator}
+    loadingIndicator={args.loadingIndicator}
+    queries={args.queries}
+  >
+    {args.children}
+  </ZetkinQuery>
+);
+
+const mockLoadingQuery = mockUseQueryResult('loading');
+const mockErrorQuery = mockUseQueryResult('error');
+const mockSuccessQuery = mockUseQueryResult('success');
+
+export const loading = Template.bind({});
+loading.args = {
+  queries: { mockLoadingQuery },
+};
+
+export const customLoading = Template.bind({});
+customLoading.args = {
+  loadingIndicator: <LinearProgress />,
+  queries: { mockLoadingQuery },
+};
+
+export const error = Template.bind({});
+error.args = {
+  errorIndicator: <Typography>THERE WAS AN ERROR!!!</Typography>,
```

Comment:
This is a custom error, there's also a default error. Maybe call this custom error? Or make one for default error